sympy/physics/control/tests/test_lti.py

```diff
@@ -163,6 +164,68 @@ def test_TransferFunction_functions():
     assert SP4.subs({a0: -1, a1: -7}).evalf() == expect4
     assert expect4_.evalf() == expect4
 
+    # Zero-frequency (or DC) gain.
+    assert tf0.dcgain() == 1
+    assert tf1.dcgain() == Rational(3, 5)
+    assert SP2.dcgain() == 0
+    assert expect4.dcgain() == -1
+    assert expect2_.dcgain() == 0
+    assert TransferFunction(1, s, s).dcgain() == oo
+
+    # Poles of a transfer function.
+    tf_ = TransferFunction(x**3 - k, k, x)
+    _tf = TransferFunction(k, x**4 - k, x)
+    TF_ = TransferFunction(x**2, x**10 + x + x**2, x)
+    _TF = TransferFunction(x**10 + x + x**2, x**2, x)
+    assert G1.pole() == [I, I, -I, -I]
+    assert G2.pole() == []
+    assert tf1.pole() == [-5, 1]
+    assert expect4_.pole() == [s]
+    assert SP4.pole() == [-a0*s]
+    assert expect3.pole() == [-0.25*p]
+    assert str(expect2.pole()) == str([0.729001428685125, -0.564500714342563 - 0.710198984796332*I, -0.564500714342563 + 0.710198984796332*I])
+    assert str(expect1.pole()) == str([-0.4 - 0.66332495807108*I, -0.4 + 0.66332495807108*I])
+    assert _tf.pole() == [k**(Rational(1, 4)), -k**(Rational(1, 4)), I*k**(Rational(1, 4)), -I*k**(Rational(1, 4))]
+    assert TF_.pole() == [CRootOf(x**9 + x + 1, 0), 0, CRootOf(x**9 + x + 1, 1), CRootOf(x**9 + x + 1, 2), \
+        CRootOf(x**9 + x + 1, 3), CRootOf(x**9 + x + 1, 4), CRootOf(x**9 + x + 1, 5), CRootOf(x**9 + x + 1, 6), \
+        CRootOf(x**9 + x + 1, 7), CRootOf(x**9 + x + 1, 8)]
+    raises(ValueError, lambda: TransferFunction(x**2, a*x**10 + x + x**2, x))
+
+    # Stability of a transfer function.
+    q, r = symbols('q, r', negative=True)
+    t = symbols('t', positive=True)
+    TF_ = TransferFunction(s**2 + a0 - a1*p, q*s - r, s)
+    stable_tf = TransferFunction(s**2 + a0 - a1*p, q*s - 1, s)
+    stable_tf_ = TransferFunction(s**2 + a0 - a1*p, q*s - t, s)
+
+    assert not G1.is_stable
+    assert G2.is_stable
+    assert not tf1.is_stable   # as one pole is +ve, and the other is -ve.
+    assert not expect2.is_stable
+    assert expect1.is_stable
+    assert stable_tf.is_stable
+    assert stable_tf_.is_stable
+    assert not TF_.is_stable
+    assert not expect4_.is_stable   # no assumption provided for the only pole 's'.
+    assert not SP4.is_stable
```

Comment:
These should test `is True`, `is False`, and `is None`, otherwise the test does not distinguish `False` and `None`.